debian/libxpum-dev.install

```diff
@@ -1,3 +1,2 @@
-usr/lib/xpu-smi
 usr/include
```

Comment:
I was always told that it's important to include the individual header files instead of a the whole directory...
is this true?
```
./usr/include/xpum_api.h
./usr/include/xpum_structs.h
```

Reply:
Is the package installing a whole directory of unnecessary files? If we don't want them, we can remove them, but I'm more inclined to install everything to avoid unintentionally breaking some functionality. That leaves the upstream devs the ability to change what gets installed without everything getting broken

Reply:
its only those headers files in the directory, but from a pkging level it helps to see whats in the dev pkg

Reply:
the build will break if there r extra files and not specified anywhere

Reply:
either way, your call, the PR is ready to merge